tests/test_boostrap.py

```diff
@@ -16,6 +16,7 @@
 # Globals
 NODE_NAME = "my_node"
 ACCT_NAME = "myaccount"
+ACCT_NAME2 = "myaccount2"
 
 
 def test_booststrap():
```

Comment:
I did a little bit of testing myself to ensure that my code suggestions were correct (or at least reasonable) and I think we should do some more robust testing here.

Currently we are testing a single positive path and I imagine we should probably test more positive paths and a some negative ones, too.

Reply:
I would be against adding another scenarios for `--bootstrap-system-full` on system level (which basically starts up the container and runs several transactions to prepare the environment), because each scenario would consume another 90 seconds.
Potentially, we might add unit tests with mocking. But in our case it is usually a wrapper for a `cleos` command, so `cleos` would tell us if there is any error. From what I see, the only thing that might be tested are `ifs` related to checking if there are a proper number of arguments.
Do you have any specific tests on your mind?

Reply:
My thought is that we haven't tested the CLI as well as we should. We should be running these as regression tests. During development, we can run the subset of tests we need and then when we have wrapped everything up, we run the full test. It's okay if that test takes 10 minutes.

For negative path testing, what happens if we send the wrong or bad CLI options? Nothing? Quiet failure? That's something we should check for.

Reply:
Thanks for getting `--bootstrap-system` back in. That was my big complaint. Improving the tests is more of a nice to have than a need to have. I'll sign off with tests as is.